core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -1,6 +1,7 @@
 <div  class="oppia-drop-area" ng-if="showDropArea">
   <div class="oppia-blur-background"></div>
-  <div class="oppia-drop-area-message">Drop your file here!</div>
+  <div class="oppia-drop-area-message" ng-if="!userIsGuest && !translationIsDisallowed">Drop your file here!</div>
```

Comment:
You could use scope.dropAreaIsShown in the ng-if?

Reply:
Yeah so scope.dropAreaIsShown 
and replace that in the markup with scope variable.

Reply:
ok should be done now

Reply:
Doesn't look like it...?